packages/cf-component-wizard/example/basic/component.js

```diff
@@ -0,0 +1,96 @@
+const React = require('react');
+const {render} = require('react-dom');
+const Wizard = require('../../src/index');
+const {Button} = require('cf-component-button');
+
+const {
+  GatewayDest,
+  GatewayProvider
+} = require('react-gateway');
+
+class Component extends React.Component {
```

Comment:
I could be wrong but shouldn't we import the component and use it in examples instead of re-implementing?